internal/builders/generic/README.md

```diff
@@ -159,19 +161,15 @@ jobs:
 
 ### Workflow Inputs
 
-The builder workflow
-[.github/workflows/generator_generic_slsa3.yml](.github/workflows/generator_generic_slsa3.yml) accepts
-the following inputs:
+The [builder workflow](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_generic_slsa3.yml) accepts the following inputs:
```

Comment:
Let's change "builder workflow" to "generic workflow" here since that's the wording we're using elsewhere.
```suggestion
The [generic workflow](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_generic_slsa3.yml) accepts the following inputs:
```